server.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
```

Comment:
I wonder if we need server.js file at all in the frontend

Reply:
I added it only to generate the graph so I can see if it works.